x/faucet/client/cli/tx.go

```diff
@@ -61,39 +61,62 @@ func GetCmdRequest(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 
-			amount, err := sdk.ParseCoins(args[0])
+			infos, err := cliCtx.Keybase.List()
 			if err != nil {
 				return err
 			}
 
-			var recipient sdk.AccAddress
-			if len(args) == 1 {
-				recipient = cliCtx.GetFromAddress()
+			var faucet, recipient sdk.AccAddress
+
+			// accepts bech32 or acct name for faucet
+			if len(args[0]) < 42 {
+				for _, info := range infos {
+					if args[0] == info.GetName() {
+						faucet = info.GetAddress()
+					}
+				}
 			} else {
-				recipient, err = sdk.AccAddressFromBech32(args[1])
+				faucet, err = sdk.AccAddressFromBech32(args[0])
+				if err != nil {
+					return err
+				}
 			}
 
-			if err != nil {
-				fmt.Println("failed to create acc address")
-				return err
+			// accepts bech32 or acct name for recipient
+			if len(args[1]) < 42 {
+				for _, info := range infos {
+					if args[1] == info.GetName() {
+						recipient = info.GetAddress()
+					}
```

Comment:
we should use a util function, afaik this is used in other parts too

Reply:
i see it used as an unexported fn in the cosmos-sdk methods. any pointers on where we use it in our code (or our implementation of that util method)?